src/utils.js

```diff
@@ -30,11 +30,26 @@ export function windowOpen(url, { name, height = 400, width = 550 }) {
     chrome: 'yes',
   };
 
-  return window.open(
+  const shareDialog = window.open(
     url,
     platform.name === 'IE' && parseInt(platform.version, 10) < 10 ? '' : name,
     Object.keys(config).map(key => `${key}=${config[key]}`).join(', ')
   );
+
+  const interval = window.setInterval(() => {
```

Comment:
Good to know :+1: 

One more thing then -  could you run `setInterval` only if `closeCallback` param was provided? Does it make sens?

```js
if (closeCallback) {
  const interval = window.setInterval(() => {
    try {
      if (shareDialog === null || shareDialog.closed) {
        window.clearInterval(interval);
        closeCallback(shareDialog);
      }
    } catch (e) {
      console.log(e);
    }
  }, 1000);
```